symbolicationWebService.py

```diff
@@ -125,42 +159,24 @@ def do_POST(self):
         return
 
       LogDebug("Request body: " + requestBody)
-      rawRequest = json.loads(requestBody)
 
-      request = SymbolicationRequest(gSymFileManager, rawRequest)
-      if not request.isValidRequest:
+      response = yield gPool.submit(processSymbolicationRequest, requestBody)
+      if response is None:
         LogDebug("Unable to parse request")
```

Comment:
Ok, can you add the IP to SymbolicationRequest and have it included in the symbolicationRequest.py messages where it makes sense?